### DISCUSSION TITLE
Card blocks without text and empty space

### DISCUSSION THREAD

**Discussion by taszap** [GitbookIO/community#155] [Feature Requests]
### What's your feature/idea?

I would like to create cards without any text underneath. Currently, if I don't feel the text a big blank space shows underneath. 

### What's your desired outcome?

My cards include relevant text in the cover image, I don't need separate text or empty space. 
I would like to enable/disable this feature. 

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

**Comment by Gomilu** [comment]
I agree
I would like to add being able to have a picture & simply write over it same as we see https://github.com/GitbookIO/community/discussions here on the top
